blockly-ws-search/main.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ *
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Example of using WorkspaceSearch with Blockly.
+ * @author kozbial@gmail.com (Monica Kozbial)
+ */
+
+import WorkspaceSearch from './workspace_search.js';
+
+const options = {
+  comments: true,
+  collapse: true,
+  disable: true,
+  maxBlocks: Infinity,
+  oneBasedIndex: true,
+  readOnly: false,
+  //rtl: false,
+  scrollbars: true,
+  trashcan: true,
+  //toolbox: null,
+  //horizontalLayout: false,
+  //toolboxPosition: 'start',
```

Comment:
We should probably remove the commented options.

Reply:
Done